src/components/MenuItem.jsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from 'grommet'
+import { normalizeColor } from 'grommet/utils'
+import styled from 'styled-components'
+
+const Item = styled(Box)`
+  font-size: ${({ theme }) => theme.text.small.size};
```

Comment:
Would it be possible to have something other than small text size here?

Reply:
Can be set directly on the child, to anything we like, this is the default.